app/javascript/components/App.jsx

```diff
@@ -1,15 +1,36 @@
 import React, { Component } from 'react';
 import Home from './Home';
+import SignUp from './SignUp';
+import SignIn from './SignIn';
+import Axios from 'axios';
 import {HashRouter as Router, Route, NavLink, Switch} from 'react-router-dom'
 
 class App extends Component {
+  handleSignOut = (e) =>{
+    e.preventDefault();
+    Axios({
+      method: 'delete',
+      url: '/auth/sign_out',
+      data: JSON.parse(sessionStorage.user)
+    }).then(function (response) {
+        window.location = "/"
```

Comment:
this is not 'React way' of redirecting. please investigate

Reply:
need to do redirecting in the controller? but we use devise-token-auth and now we do not have a controller, it is built-in.

So need create controller, do override. And this all for only rediracting. I think it's bad way